.github/workflows/railway-status-check.yml

```
@@ -0,0 +1,113 @@
+      # 1. Checkout code
+      # ======================================================================
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      # ======================================================================
+      # 2. Check application health
+      # ======================================================================
+      - name: 🏥 Check application health
+        id: health-check
+        continue-on-error: true
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:13:28: Double quote to prevent globbing and word splitting

(shellcheck)